### PR TITLE
my solution to the challenges

### PR DESCRIPTION
**Pull request by vikyij**

### CODE REVIEW

File: src/brackets/index.js [hunk 1]
```diff
@@ -4,6 +4,26 @@
...
+function isValid(str) {
+    let brackets = '()[]{}'
+    let stack = []
+
+    for (let bracket of str) {
+        let bracketIndex = brackets.indexOf(bracket)
+
+        if(bracketIndex === -1) {
+            continue
+        }
```
Comment (bondz) [inline src/brackets/index.js:16]: What is the implication of this? Wouldn't that mean the string is not valid and you can return?

File: src/roman-numerals/index.js [hunk 2]
```diff
@@ -4,6 +4,33 @@
...
+    'V': 5,       
+    'X': 10,       
+    'L': 50,
+    'C': 100,        
+    'D': 500,       
+    'M': 1000
+}
+
+function romanToDecimal(roman) {
+   let array = roman.split('')
```
Comment (bondz) [inline src/roman-numerals/index.js:18]: This can be `const` too.

File: src/transpose/index.js [hunk 3]
```diff
@@ -4,6 +4,8 @@
  * @param {number[]} array The array to transpose
  * @returns {number[]} The transposed array
  */
-function transpose(array) {}
+function transpose(array) {
+    return array[0].map((col, i) => array.map(row => row[i]));
```
Comment (bondz) [inline src/transpose/index.js:8]: Nice and short.

Comment (vikyij) [reply src/transpose/index.js:8]: thank you

File: src/brackets/index.js (hunk 1 above)
Comment (vikyij) [reply src/brackets/index.js:16]: yeah

**Pull request merged**